app/src/main/java/com/android/partagix/ui/screens/InventoryCreateOrEditItemScreen.kt

```diff
@@ -128,19 +140,30 @@ fun InventoryCreateItem(
 
             Row(modifier = modifier.fillMaxWidth()) {
               Box(modifier = modifier.fillMaxWidth(.5f).padding(end = 8.dp)) {
-                uiCategory = Category("", DropDown("Category", CategoryItems))
+                uiCategory =
+                    Category(
+                        "", DropDown(uiCategory.name, CategoryItems)) // TODO: get category id(?)
```

Comment:
This comment can be removed, the Category.id is recovered directly by the viewmodel.

Reply:
okay I didn't checked that sry.
Thx!